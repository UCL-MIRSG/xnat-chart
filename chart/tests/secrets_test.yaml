templates:
  - templates/secrets.yaml

tests:
  - it: secrets manifest should match snapshot
    set:
      imageCredentials.enabled: true
    asserts:
      - matchSnapshot: {}
