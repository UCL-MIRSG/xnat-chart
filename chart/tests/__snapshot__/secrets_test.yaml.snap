secrets manifest should match snapshot:
  1: |
    apiVersion: v1
    data:
      .dockerconfigjson: eyJhdXRocyI6eyIiOnsidXNlcm5hbWUiOiIiLCJwYXNzd29yZCI6IiIsImVtYWlsIjoiIiwiYXV0aCI6Ik9nPT0ifX19
    kind: Secret
    metadata:
      name: image-pull-secret
    type: kubernetes.io/dockerconfigjson
